docs/quickstart/cross-chain-transfers.md

```diff
@@ -0,0 +1,480 @@
+- A 6-character base-32 encoded error correction code using the BCH algorithm.
+
+AvalancheJS has address encoding for the following networks
+
+```text
+0 - X-custom19rknw8l0grnfunjrzwxlxync6zrlu33yeg5dya
+1 - X-avax19rknw8l0grnfunjrzwxlxync6zrlu33y2jxhrg
+2 - X-cascade19rknw8l0grnfunjrzwxlxync6zrlu33ypmtvnh
+3 - X-denali19rknw8l0grnfunjrzwxlxync6zrlu33yhc357h
+4 - X-everest19rknw8l0grnfunjrzwxlxync6zrlu33yn44wty
+5 - X-fuji19rknw8l0grnfunjrzwxlxync6zrlu33yxqzg0h
+1337 - X-custom19rknw8l0grnfunjrzwxlxync6zrlu33yeg5dya
+12345 - X-local19rknw8l0grnfunjrzwxlxync6zrlu33ynpm3qq
+```
+
+### Edit The Amount Of AVAX You Want To Send:
+
+By default the scripts send the wallet's entire AVAX balance:
+
+When using Avalanche Network Runner, we import our [Pre-Generated Private Key](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts#L30) to our [`xKeychain`](https://github.com/ava-labs/avalanchejs/blob/46ce89f395133702320a77cba4bb9cb818b48fe8/examples/avm/buildExportTx-cchain-avax.ts#L31) to obtain signers. This means `X-custom18jma8ppw3nhx5r4ap8clazz0dps7rv5u9xde7p` is the `balance` holder and will pay the `fee` to export AVAX.
+
+```js
+const balance: BN = new BN(getBalanceResponse.balance);
+const amount: BN = balance.sub(fee);
+```
+
+Change the amount by creating a new _BN_ variable: `value` and assigning it a string value `"10000000000000000"` (.01 AVAX) as an example.
+
+```js
+const value: BN = new BN("10000000000000000");
+const amount: BN = value.sub(fee);
+```
+
+We must pass the value in WEI format. For reference, 10 \*\* 18 WEI = 1 AVAX
+
+You can use the [snowtrace unit converter](https://snowtrace.io/unitconverter) to view more unit conversions
+
+### Step 5 - Run the Export and Import scripts
+
+Between X/P/C chains there is shared memory. First, tokens are exported from the source chain to the shared memory, then imported by the destination chain.
+
+To complete a transfer from the X-Chain to the C-Chain , we must run both the Export and Import scripts.
+
+By default, the scripts use the AvalancheJS constant, [`DefaultLocalGenesisPrivateKey`](https://github.com/ava-labs/avalanchejs/blob/master/examples/avm/buildExportTx-cchain-avax.ts#L30) as `privKey` to sign transactions.
+
+```js
+const privKey: string = `${PrivateKeyPrefix}${DefaultLocalGenesisPrivateKey}`;
+```
+
+The bech32 address which derived from this private key is : `X-custom18jma8ppw3nhx5r4ap8clazz0dps7rv5u9xde7p`
+
+That key is pre-funded on the local network when using Avalanche Network Runner and has the following AVAX in the genesis block/vertex of each chain on the primary subnet:
+
+- X-Chain: 300000000000000000 nAVAX
+
+- P-Chain: 30000000000000000 nAVAX
+
+- C-Chain: 50000000000000000000000000 nAVAX
+
+```zsh
+avalanchejs $ ts-node examples/avm/buildExportTx-cchain-avax.ts
+Success! TXID: Rgg2412kaczRYC3taasvG6bYoqG7tBQG6WfacNdumKDKsVWpF
+avalanchejs $ ts-node examples/evm/buildImportTx-xchain.ts
```

Comment:
buildImportTx-xchain.ts was never introduced until now. Please mention that above changes should be applied to both buildExportTx-cchain-avax.ts and buildImportTx-xchain.ts before running them.